src/main/java/application/AbstractBaseController.java

```diff
@@ -289,6 +298,15 @@ public ExceptionResponseBean handleError(FormplayerHttpRequest req, Exception ex
         return new ExceptionResponseBean(exception.getMessage(), req.getRequestURL().toString());
     }
 
+    private void incrementDatadogCounter(String metric, FormplayerHttpRequest req) {
+        datadogStatsDClient.increment(
+                metric,
+                "domain:" + req.getDomain(),
+                "user:" + req.getCouchUser().getUsername(),
```

Comment:
assumption here is that if we ever have a `FormplayerHttpRequest` then we have a user. let me know if you think that's a bad assumption